src/main/java/org/broadinstitute/consent/http/models/dto/DataOwnerCase.java

```diff
@@ -93,7 +93,7 @@ public String getAlias() {
     }
 
     public void setAlias(Integer alias) {
-        this.alias = DatasetUtil.parseAlias(alias);
+        this.alias = "DUOS-" + StringUtils.leftPad(alias.toString(), 6, "0");
 
     }
```

Comment:
See other comment, we should be getting this from the Dataset, not copying the content of the method.